### PR TITLE
[Pending] Feature/add logger

### PR DESCRIPTION
**Pull request by MizukiSonoko**
## What this function?
Logger function like this
```c++
  log.info(R"( dP  888888ba   .88888.  dP     dP   .d888888 )");
  log.info(R"( 88  88    `8b d8'   `8b 88     88  d8'    88 )");
  log.info(R"( 88 a88aaaa8P' 88     88 88aaaaa88a 88aaaaa88a)");
  log.info(R"( 88  88   `8b. 88     88 88     88  88     88 )");
  log.info(R"( 88  88     88 Y8.   .8P 88     88  88     88 )");
  log.info(R"( dP  dP     dP  `8888P'  dP     dP  88     88 )");
  log.info(R"( ---------o=========================o---------)");
  log.info(R"(         = 分散台帳Application いろは =         )");
  log.info(R"( ---------o=========================o---------)");
```
## Why you implements?
To show status to developer, of course used by debug

## What does module use it and how?
all module (main, consensus, network, validation, ordering...) 

## P.S
Maybe this pull request will fat, so I create  sub pull request.

### CODE REVIEW

File: irohad/validation/stateless/validator.cpp [hunk 1]
```diff
@@ -19,13 +19,18 @@ limitations under the License.
...
+      log.debug("Does transaction have a signature?:{}", tx.header().signature_size() != 0);
+      log.debug("Is creator publicKey correct?:{} (length:{})", tx.body().creator_pubkey().size() == 32, tx.body().creator_pubkey().size());
       return
         tx.header().created_time() <= iroha::time::now64() && // 過去に作られたTxか // TODO: consider when to ignore transactions for being too old
-        tx.header().signature_size() != 0                          && // 電子署名は含まれているか
+        tx.header().signature_size() != 0                  && // 電子署名は含まれているか
         // TODO: calculate hash
         // TODO: verify the signature for each signature in the header
-        tx.body().creator_pubkey().size() == 32;                     // 公開鍵は32byteか ToDo configurable
+        tx.body().creator_pubkey().size() == 44;          // 公開鍵は32byteか ToDo configurable
```
Comment (lebdron) [inline irohad/validation/stateless/validator.cpp]: The numbers should probably come from iroha::crypto::ed25519 namespace

Comment (MizukiSonoko) [inline irohad/validation/stateless/validator.cpp]: ok I will fix 👍

Comment (Warchant) [inline irohad/validation/stateless/validator.cpp]: And now from `libs/common/types.h`

**Comment by MizukiSonoko** [comment]
In now, It's not needed... 😢

**Comment by motxx** [comment]
Please do not delete this branch. It will be used in a few days.

**Comment by MizukiSonoko** [comment]
Of course, I will use it 
I'll be back......
 ⛓  👍  ⛓
🔥🔥🔥🔥🔥

**Pull request closed without merging**